### PR TITLE
Fix vite manifest not found

### PR DESCRIPTION
**Pull request by DeMiro5001**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by DeMiro5001** [comment]
!testme

**Comment by DeMiro5001** [comment]
!testme

**Comment by DeMiro5001** [comment]
!testme

**Comment by DeMiro5001** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -42,13 +42,17 @@ ynh_script_progression "Installing app with Composer..."
...
+
 #=================================================
 # BUILDING
 #=================================================
 ynh_script_progression "configuring $app..."
 
 pushd "$install_dir"
-	"php$php_version" artisan key:generate -n --force --env
+	 npm install 2>&1
+         npm run build 2>&1
```
Comment (ericgaspar) [inline scripts/install:54]:
```suggestion
	 ynh_hide_warnings npm install
         ynh_hide_warnings npm run build
```

File: scripts/upgrade [hunk 2]
```diff
@@ -46,7 +47,11 @@ ynh_script_progression "Configuring $app..."
 ynh_composer_exec dump-autoload
 
 pushd "$install_dir"
-	#"php$php_version" artisan key:generate -n --force --env
+
+         npm install 2>&1
+         npm run build 2>&1
```
Comment (ericgaspar) [inline scripts/upgrade:52]:
```suggestion
         ynh_hide_warnings npm install
         ynh_hide_warnings npm run build
```

File: scripts/_common.sh [hunk 3]
```diff
@@ -5,3 +5,4 @@
 #=================================================
 
 composer_version="2.7.7"
+nodejs_version=22
```
Comment (ericgaspar) [inline scripts/_common.sh:8]:
```suggestion
nodejs_version="22"
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

**Comment by DeMiro5001** [comment]
!testme

**Comment by orhtej2** [comment]
While this very likely gets Lychee running these steps are performed on their CI already, I feel like changing the source artifact to `Lychee.zip` instead of tag tarball would get the app up and running.

**Comment by orhtej2** [comment]
Testing here: #136

**Comment by DeMiro5001** [comment]
> While this very likely gets Lychee running these steps are performed on their CI already, I feel like changing the source artifact to `Lychee.zip` instead of tag tarball would get the app up and running.

I prefer your solution. And it takes less time according to bullseye Ci (since it runs one job at a time, so other jobs won't slow it).

**Comment by orhtej2** [comment]
Superseded by #136

**Pull request closed without merging**